client/src/components/ProjectSamplesTable.js

```diff
@@ -78,7 +78,12 @@ export const ProjectSamplesTable = ({
     { Header: 'Technology', accessor: 'technologies' },
     {
       Header: 'Other Modalities',
-      accessor: ({ modalities }) => modalities || 'N/A'
+      accessor: ({ modalities }) =>
+        modalities.map((modality, i) => (
+          <>
+            {i ? ' ,' : ''} {modality}
+          </>
+        )) || 'N/A'
```

Comment:
Thanks for doing this so fast!

As I'm not really sure of the data structures here, I'm not sure whether we can just use `modalities.join(', ')` to do this more simply as we do for the additional metadata? Are the `<>` & `</>` required?

Either way, we should reverse the comma space order, as suggested below:
```suggestion
      accessor: ({ modalities }) =>
        modalities.map((modality, i) => (
          <>
            {i ? ',' : ''} {modality}
          </>
        )) || 'N/A'
```

Reply:
Revised the spacing (oops), thank you Josh!

**Note:** The `<></>`(short form of `<React.Fragment>`) is called [fragments](https://reactjs.org/docs/fragments.html) and is the way to properly render [JSX](https://reactjs.org/docs/introducing-jsx.html) in React UI :)